prusa/connect/printer/cameras.py

```diff
@@ -242,6 +286,30 @@ def camera_id(self):
         """Returns this camera's ID"""
         return self._driver.camera_id
 
+    @property
+    def fingerprint(self):
+        """Returns the camera ID as a fingerprint for connect"""
+        return make_fingerprint(self.camera_id)
+
+    @property
+    def is_registered(self):
+        """Is the camera registered to connect?"""
+        return self._token is not None
+
+    @property
+    def token(self):
+        """Return the camera token"""
+        return self._token
+
+    def register_handler(self, token):
+        """Saves the camera token"""
+        self._token = token
+        self.save()
+
+    def unregister(self):
+        """Removes the camera token"""
+        self._token = None
```

Comment:
There are some problems:
* register save the configuration, but unregistrer not
* that functionality is fine for failed_cb but not for manual unregistration cameras from connect

Reply:
There's no manual unregistration yet. I just forgot to put the save in the callback. I'll leave it as is. We'll figure out the saving mechanism when adding the unregister API endpoint